inference/deepdive-model

```diff
@@ -0,0 +1,242 @@
+                else
+                    cd "$DEEPDIVE_APP"
+                    src="snapshot/model/$What/$Name"
+                    [[ -e $src ]] ||
+                        error "\`deepdive keep $What $Name\` has never run or $Name got removed"
+                fi
+                case $Command in
+                    reuse)
+                        case $What in
+                            factorgraph)
+                                cd "$DEEPDIVE_APP"
+                                # TODO use (relative) symlink or cp -al instead of duplicating with rsync?
+                                rsync -avH --delete --copy-unsafe-links "$src"/factorgraph{,.done} run/model/
+                                # TODO load all table for grounding as well
+                                ;;
+                            weights)
+                                # TODO load into a unique table, then create a view with the following name
+                                weightTableName=dd_graph_weights_reuse
+                                weightsReuseFlag="$DEEPDIVE_APP"/run/model/grounding/factor/weights.reuse
+                                deepdive create table "$weightTableName" "description:TEXT" "categories:TEXT" "weight:DOUBLE PRECISION"
+                                rm -f "$weightsReuseFlag"
+                                for weightFile in "$src"/dd_weights{,.multinomial}.tsv.bz2; do
+                                    [[ -e "$weightFile" ]] || continue
+                                    case $weightFile in
+                                        *.multinomial.tsv.bz2)
+                                            # first load the grouped data from file
+                                            deepdive create table "${weightTableName}_array" "description:TEXT" "categories::TEXT[]" "weights:DOUBLE PRECISION[]"
+                                            # TODO parallel load
+                                            deepdive load "${weightTableName}_array" "$weightFile"
+                                            # explode the group
+                                            # XXX UNNEST is postgres-specific
+                                            deepdive sql "INSERT INTO $weightTableName
+                                                SELECT description
+                                                     , UNNEST(categories) AS categories
+                                                     , UNNEST(weights)    AS weight
+                                                  FROM ${weightTableName}_array
+                                            "
+                                            deepdive sql "DROP TABLE ${weightTableName}_array"
+                                            ;;
+                                        *.tsv.bz2)
+                                            # TODO parallel load
+                                            deepdive load "$weightTableName" "$weightFile"
```

Comment:
Does this work? I ran `deepdive model weights reuse foo` on the spouse tsv example, and it I got
```
ERROR:  missing data for column "weight"
CONTEXT:  COPY dd_graph_weights_reuse, line 1: "f_has_spouse_features--word_between=D	0.723347"
```
Table `dd_graph_weights_reuse` has three columns, but the dumped file that we are trying to load only has two columns.